File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int[] answer = {Integer.MAX_VALUE, Integer.MAX_VALUE, 0, 0};
        
        for (int i=0; i<wallpaper.length; i++) {
            for (int j=0; j<wallpaper[0].length(); j++) {
                if (wallpaper[i].charAt(j) != '#') continue;
                
                if (answer[0] > i) {
                    answer[0] = i;
                }
                
                if (answer[1] > j){
                    answer[1] = j;
                }
                
                if (answer[2] < i+1) {
                    answer[2] = i+1;
                }
                
                if (answer[3] < j+1) {
                    answer[3] = j+1;
                }
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.cpp
#include <string>
#include <vector>
#include <math.h>

using namespace std;

vector<int> solution(vector<string> wallpaper) {
    vector<int> answer(4);
    
    answer[0] = answer[1] = INT32_MAX;
    
    for (int i=0; i<wallpaper.size(); i++) {
        for (int j=0; j<wallpaper[i].length(); j++) {
            char* cur = &wallpaper[i][j];
            
            if (*cur == '#') {
                answer[0] = min(answer[0], i);
                answer[1] = min(answer[1], j);
                answer[2] = max(answer[2], i+1);
                answer[3] = max(answer[3], j+1);
            }
        }
    }
    
    return answer;
}